autoload/neoformat.vim

```diff
@@ -75,20 +75,42 @@ function! s:neoformat(bang, user_input, start_line, end_line) abort
             continue
         endif
 
-        let stdin = getbufline(bufnr('%'), a:start_line, a:end_line)
+        if cmd.range
+          " Pass the entire buffer, the formatter itself takes a range.
+          let stdin = getbufline(bufnr('%'), 1, '$')
+        else
+          let lines_after = getbufline(bufnr('%'), a:end_line + 1, '$')
+          let lines_before = getbufline(bufnr('%'), 1, a:start_line - 1)
```

Comment:
It seems that `lines_after` and `lines_before`

Reply:
It looks like your comment was cutoff, what about them?

Reply:
Oops. Yeah I meant to say that it seems that `lines_after` and `lines_before` aren't used